src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -1062,6 +1062,35 @@ export default class Fms {
     }
 
     /**
+     *
+     *
+     * @for Fms
+     * @method isDeparture
+     * @return {boolean}
+     */
+    isDeparture() {
+        return this.currentPhase === FLIGHT_PHASE.APRON ||
+            this.currentPhase === FLIGHT_PHASE.TAXI ||
+            this.currentPhase === FLIGHT_PHASE.WAITING ||
+            this.currentPhase === FLIGHT_PHASE.TAKEOFF ||
+            this.currentPhase === FLIGHT_PHASE.CLIMB;
```

Comment:
two reasons; `_includes()` is slow, and this is an easy to see what the current phase must be in order for it to be considered a departure